TestFoundation/TestStream.swift

```diff
@@ -208,6 +218,75 @@ class TestStream : XCTestCase {
         XCTAssertEqual(.error, outputStream!.streamStatus)
     }
     
+    func test_streamDelegateProperty() {
+        let message: NSString = "Hello, playground"
+        let messageData: Data = message.data(using: String.Encoding.utf8.rawValue)!
+        let inputStream: InputStream = InputStream(data: messageData)
+        let outputStream: OutputStream = OutputStream.toMemory()
+        let delegate = TestStreamDelegate.init { (_, _) in }
+        inputStream.delegate = delegate
+        outputStream.delegate = delegate
+        XCTAssertNotNil(inputStream.delegate)
+        XCTAssertNotNil(inputStream.delegate)
+    }
+    
+    func test_streamDelegate() {
+        // TODO: macOS/iOS: Fix CFNetwork's dynamic bindings for `callBacks` field.
+        // TODO: Linux: Enabling delegate/scheduling tests when `CFStreamCreatePairWithSocketToHost()` implemented.
+        #if false
```

Comment:
Can this point to a specific SR and/or Radar # for follow-up work?

Reply:
I am not sure this is a bug or intended behavior. This code crashed in my Xcode. I guess SwiftFoundation fails to load CFNetwork and fill callback pointers in CFStream socket object. (Regarding SwiftFoundation target is primarily Linux and macOS Foundation works fine.)

Reply:
We can't cause SCF compiled on macOS to crash, because that's a development environment we would like contributors to use. And we should not be picking anything up from CFNetwork in that case either. :(

So something is amiss.

Reply:
I’m novice in C and unable to resolve this (don’t know how to fix dynamic loading), unfortunately. Thus I decided to disable the code and hope an expert would fix it later.
Interestingly, afaik no other member of CFReadStream cluster classes are able to use scheduling and I have to stick to socket.